### PR TITLE
curl: fix build issue on haas200, fix #1741

### LINKED ISSUE #1741: [Bug Report]: curl组件编译失败
### 联系方式

_No response_

### 您发现了什么问题

A bug happened!
运行环境:
固件版本:

### 分支

master (Default)

### 请选择您发现此问题的解决方案名称

_No response_

### 日志

_No response_

### PR DESCRIPTION
**Pull request by jinliangli**

### CODE REVIEW

**Comment by skylarCai** [comment]
感谢提交 Pull Requests！
Thanks for submitting a pull request.

**Comment by skylarCai** [comment]
Security Check : b7db1bda32d07cf16dc5b7b63a5ea21162f090f9

**Comment by skylarCai** [comment]
auto build success,number:179

**Comment by skylarCai** [comment]
autotest fail,number:1410

File: components/curl/package.yaml [hunk 1]
```diff
@@ -40,6 +40,8 @@ depends:
 build_config:
   include:
     - include
+
```
Comment (skylarCai) [inline components/curl/package.yaml:43]: LGFM

File: components/mbedtls/platform/include/platform_config.h [hunk 2]
```diff
@@ -7,7 +7,7 @@
     #include "config_yoc_alicoap.h"
 #elif defined(CONFIG_CLOUDIO_ALIMQTT)
     #include "config_yoc_alimqtt.h"
-#elif defined(CONFIG_USING_TLS)
+#elif defined(CONFIG_USING_YOC_TLS)
```
Comment (skylarCai) [inline components/mbedtls/platform/include/platform_config.h:10]: LGFM

**Review by skylarCai — COMMENTED** [review]
no security risk

File: components/mbedtls/platform/include/platform_config.h (hunk 2 above)
Comment (jinliangli) [reply components/mbedtls/platform/include/platform_config.h:10]: LGFM:  什么意思？

**Review by yanxiaoyong-ch — COMMENTED** [review]
this looks ok for me

**Pull request merged**